src/lib/isAlpha.js

```diff
@@ -17,7 +17,14 @@ export default function isAlpha(_str, locale = 'en-US', options = {}) {
     }
   }
 
-  if (locale in alpha) {
+  if (Array.isArray(locale)) {
+    const regex = locale.map(localeKey => alpha[localeKey])
+      .filter(localeRegex => !!localeRegex)
+      .map(localeRegex => localeRegex.source.replace(/\^|\$/g, ''))
```

Comment:
We may also avoid to do this `replace` (which may be expensive) by directly saving regexes without those parts
e.g.
```js
[a-z]+ 
// instead of
/^[a-z]+$/g
```

Reply:
This is done.